src/core/ui.js

```diff
@@ -87,8 +87,55 @@ function errWarn(msg, arr, butName, title) {
 
 function createWarnButton(butName, arr, title) {
   const buttonId = `respec-pill-${butName}`;
+  var numSteps = 20.0;
+  var boxElement
+  var prevRatio = 0.0;
+
+  button.addEventListener("load", function (event) {
```

Comment:
As above. Probably don't need this, you have direct access to the `respecUI` div.